journal/modernized/05-chapter-5.adoc

```diff
@@ -360,10 +363,14 @@ David Ferris
 [.signed-section-context-close]
 Wilmington, 5th Mo. 1779.
 
+[.postscript]
 P+++.+++ S. I do not think it probable that Friends will
 think the rough remarks and observations I have made,
 worth publishing; but they may possibly be of some use to my family, to whom, therefore,
 I leave them.
+//And here we are, 240 years later, publishing his remarks and observations. :) love it. 
 
 [.signed-section-signature]
 David Ferris
+
```

Comment:
This ending -- for the letter will also have to be removed, since its not a letter.